specifyweb/frontend/js_src/lib/components/Molecules/DateElement.tsx

```diff
@@ -19,14 +19,42 @@ export function DateElement({
   readonly fallback?: React.ReactNode;
   readonly flipDates?: boolean;
 }): JSX.Element {
+  const [relativeDate, setRelativeDate] = React.useState<string | undefined>(
+    date ? getRelativeDate(new Date(date)) : undefined
+  );
+
+  React.useEffect(() => {
+    let intervalValue;
```

Comment:
(optional) set default value for intervalValue to 0
the behavior won't change since setInterval's default is 0, but this will make code more clearer